backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -118,6 +119,7 @@ private async void OnIsarStatus(object? sender, MqttReceivedArgs mqttArgs)
                     _logger.LogDebug("Semaphore acquired for updating robot current mission id");
 
                     await RobotService.UpdateCurrentMissionId(robot.Id, null);
+                    await MissionScheduling.StartNextMissionRunIfSystemIsAvailable(robot);
```

Comment:
This should trigger the robot available event and not start directly from the mqtt event handler